modulefiles/gsi_orion.lua

```diff
@@ -1,20 +1,22 @@
 help([[
 ]])
 
-prepend_path("MODULEPATH", "/apps/contrib/NCEP/libs/hpc-stack/modulefiles/stack")
+prepend_path("MODULEPATH", "/work/noaa/epic-ps/role-epic-ps/miniconda3/modulefiles")
```

Comment:
Does this `prepend_path` point at the official installation all UFS and EMC apps should use?

Reply:
I have been using /work/noaa/epic-ps/hpc-stack/libs/intel/2022.1.2/modulefiles/stack, but I use an external scripted wrapper that skips all of the /ush/ and /modulefiles/ altogether.

Reply:
@natalie-perlin Confirmed this is the correct miniconda3 installation.

Reply:
Resolve with caveat noted above that g-w and library team staff need to confirm that machine specific modulefiles in this PR use officially maintained and supported modules.